samples/bookinfo/src/productpage/productpage.py

```diff
@@ -425,6 +425,8 @@ def flush(self):
 
     p = int(sys.argv[1])
     logging.info("start at port %s" % (p))
-    # Python does not work on an IPv6 only host
-    # https://bugs.python.org/issue10414
-    app.run(host='0.0.0.0', port=p, debug=True, threaded=True)
+    # Make it compatible with IPv6 if Linux
+    if sys.platform == "linux":
```

Comment:
would this break ipv4?  if not, should we do it on all platforms?

Reply:
The previous way was only working on IPv4 on any platform.
Now, it works with IPv4 and IPv6 on linux. And IPv4 on any other platform.